### DISCUSSION TITLE
I noticed that when using COMMAND_GROUPS padding is all over the place

### DISCUSSION THREAD

**Discussion by alexanderilyin** [ewels/rich-click#218] [Q&A]
![image](https://github.com/user-attachments/assets/446aae4c-2e31-4a08-b805-9e35578eea77)

I wonder if this is by design or it's a place for improvement.

**Comment by dwreeves** [comment]
place for improvement. this is a goal for 1.10. we may just set expand=False for the table, but we'll see.
